### DISCUSSION TITLE
Any other way to retrieve Admin Password?

### DISCUSSION THREAD

**Discussion by hjhirsch** [GerryDazoo/Slinger#300] [General]
Hi, thanks for this....  Unfortunately I was unable to obtain Admin password from URL even though I accessed once via the link in time.

I was wondering if there is another way to skin this cat?
Today while I was on the same network as Slingbox (11/14/2022) via my Samsung Note, I was able to control Slingbox and stream locally to my phone but then I messed up and tried to access remotely,  after that I was no longer able to access or stream again.

Do you think I can find the admin password somewhere on my Android phone (in some cache), other phones I have not used in a while?  Perhaps if I reload the Sling Desktop software (I remember a special ID when I first installed) or look at an older XP computer that used to access my sling?

Grasping at strawas trying to keep my Sling alive....  Thanks in advance!

**Comment by sp222222** [comment]
someone has figured out the 350/500 passwords...... stay tuned.

**Reply by hjhirsch** [reply]
Can't wait for update, thanks again.  Please let me know if my experience of using Sling on local network can help in any way.

**Reply by socelillyhammer** [reply]
any updates on this?

**Reply by socelillyhammer** [reply]
see my recent post! you can get your pw!  I just did it with mine.  I FOUND OUT HOW TO GET THE PW TO 500! @hjhirsch

**Comment by hjhirsch** [comment]
BTW, I also accessed my sling player via the Amazon FireTV app (loaded on at least 5 FireTV sticks).  I won't dare try to access my SB in case there is a way to obtain password from within the FireTV stick.
